cornice/service.py

```diff
@@ -413,17 +413,21 @@ def cors_enabled(self):
     def cors_enabled(self, value):
         self._cors_enabled = value
 
-    @property
-    def cors_supported_headers(self):
+    def cors_supported_headers(self, method=None):
```

Comment:
I updated the PR, and generalized it to cors_support_credentials.

It looks weird when method param is omitted, but it's now consistent